tango_ros_common/tango_ros_native/src/tango_ros_node.cpp

```diff
@@ -446,6 +473,53 @@ Tango3DR_Status TangoRosNode::TangoSetup3DRConfig() {
         << use_floorplan << " error: " << result;
     return result;
   }
+
+  node_handle_.param(TANGO_3DR_USE_SPACE_CLEARING_PARAM_NAME,
+                     t3dr_resolution_, TANGO_3DR_DEFAULT_USE_SPACE_CLEARING);
```

Comment:
this explains so much :blush: , I already wondered why playing with the parameters did not really change anything